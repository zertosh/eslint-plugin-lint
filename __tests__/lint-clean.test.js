'use strict';

const path = require('path');
const child_process = require('child_process');

test('eslint-plugin-lint is lint clean', () => {
  const eslintRet = child_process.spawnSync(
    'node_modules/.bin/eslint',
    [
      '.',
    ],
    {
      cwd: path.join(__dirname, '..'),
      encoding: 'utf8',
    }
  );
  expect(eslintRet.error).toBeUndefined();
  expect(eslintRet.stderr).toMatchSnapshot();
  expect(eslintRet.stdout).toMatchSnapshot();
  expect(eslintRet.status).toEqual(0);
});
