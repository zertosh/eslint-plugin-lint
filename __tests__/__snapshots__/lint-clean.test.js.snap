// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`eslint-plugin-lint is lint clean 1`] = `""`;

exports[`eslint-plugin-lint is lint clean 2`] = `""`;
